scrapper/pom.xml

```diff
@@ -3,106 +3,151 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>BookFinder</artifactId>
-        <groupId>pl.codebrothers</groupId>
-        <version>1.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.5.RELEASE</version>
```

Comment:
Maybe try to import Spring boot version from properties?

Reply:
Thank you, it will be improved in the next pull request